### PR TITLE
[RTM]search for reportlets from directory where subject id was found not r…

### PR DESCRIPTION
**Pull request by rwblair**
…oot of reports dir

### CODE REVIEW

File: fmriprep/viz/reports.py [hunk 1]
```diff
@@ -142,8 +142,9 @@ def run_reports(out_dir):
         dir = root.split('/')[-1]
         try:
             subject = re.search('^(?P<subject_id>sub-[a-zA-Z0-9]+)$', dir).group()
+            print(subject)
```
Comment (chrisfilo) [inline fmriprep/viz/reports.py]: left over debug print

**Comment by chrisfilo** [comment]
I still cannot see the final aggregated HTML file in the CircleCi artifacts. What am I missing?

**Comment by rwblair** [comment]
this commit also in #242 will merge from there once reports in circle are showing up.

**Comment by chrisfilo** [comment]
thanks!

**Pull request closed without merging**